Makefile-test.mk

```diff
@@ -151,6 +154,15 @@ run-test-tas-e2e-%: FORCE
 		./hack/e2e-test.sh
 	$(PROJECT_DIR)/bin/ginkgo-top -i $(ARTIFACTS)/$@/e2e.json > $(ARTIFACTS)/$@/e2e-top.yaml
 
+run-test-queuename-e2e-%: K8S_VERSION = $(@:run-test-queuename-e2e-%=%)
+run-test-queuename-e2e-%: FORCE
+	@echo Running e2e for k8s ${K8S_VERSION}
+	E2E_KIND_VERSION="kindest/node:v$(K8S_VERSION)" KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) CREATE_KIND_CLUSTER=$(CREATE_KIND_CLUSTER) \
+		ARTIFACTS="$(ARTIFACTS)/$@" IMAGE_TAG=$(IMAGE_TAG) GINKGO_ARGS="$(GINKGO_ARGS)" \
+		JOBSET_VERSION=$(JOBSET_VERSION) \
```

Comment:
Do we need to prepare jobset?

Reply:
Not for this PR. I left it there with the idea of growing the suite with other custom configurations as Michal suggested. However, I agree there's no need to include it in this PR